pycbc/filter/resample.py

```diff
@@ -292,6 +295,50 @@ def highpass(timeseries, frequency, filter_order=8, attenuation=0.1):
     return TimeSeries(lal_data.data.data, delta_t = lal_data.deltaT,
                       dtype=timeseries.dtype, epoch=timeseries._epoch)
 
+def lowpass(timeseries, frequency, filter_order=8, attenuation=0.1):
```

Comment:
Maybe this change should be split to a different PR? It seems fairly independent from the rest of the PR.

Reply:
I think that makes sense - I'll make a separate PR